kubectl-rook-ceph.sh

```diff
@@ -226,14 +227,202 @@ function path_cm_rook_ceph_operator_config() {
 }
 
 ####################################################################################################
-# 'kubectl rook-ceph mon-endpoints' commands
+# 'kubectl rook-ceph mons' commands
 ####################################################################################################
 
+function run_mons_command () {
+  if [ "$#" -ge 1 ] && [ "$1" = "reset-quorum" ]; then
+    shift # remove the subcommand from the front of the arg list
+    run_reset_quorum "$@"
+  else
+    fetch_mon_endpoints "$@"
+  fi
+}
+
 function fetch_mon_endpoints() {
   end_of_command_parsing "$@" # end of command tree
   KUBECTL_NS_CLUSTER get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data' | tr -d '"' | tr -d '=' | sed 's/[A-Za-z]*//g'
 }
 
+wait_for_pod_of_deployment_to_be_running() {
+  deployment=$1
+  echo -e "\nWaiting for the pod from deployment \"$deployment\" to be running"
```

Comment:
```suggestion
  info_msg " Waiting for the pod from deployment \"$deployment\" to be running"
```

in other places, for logging, use this 
```
function info_msg() {
  echo -e "${INFO_PREFIX}" "$@"
}

function warn_msg() {
  echo -e "${WARNING_PREFIX}" "$@"
}

function error_msg() {
  echo -e "${ERROR_PREFIX}" "$@"
}
```